docs/UserGuide.md

```diff
@@ -147,6 +147,21 @@ Examples:
 - `list` followed by `delete 2` deletes the 2nd person in the address book.
 - `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding Notes to an existing contact: `addnotes`
+
+Adds a note to a person in the address book.
+
+Format: `addnotes INDEX n/[NOTES]`
+
```

Comment:
Indicate that `INDEX` must be positive and within the range of the list like in the `delete` section above